src/genomic_features/_core/filters.py

```diff
@@ -211,6 +220,99 @@ def required_tables(self) -> set[str]:
         return {"gene"}
 
 
+class SeqFilter(AbstractFilterEqualityExpr):
```

Comment:
One minor failure mode of this filter is that you get a rather obscure error if you pass chromosomes as integers instead of strings:
```pytb
>>> hsapiens108.genes(cols=["tx_id", 'gene_name', 'seq_name'], filter=gf.filters.SeqFilter(21))
IbisTypeError: argument passes none of the following rules: tuple_of(value(None,),), column(value(None,),), value(,), value(,)
``` 
It would be good to specify with error or in docstrings that the input should be strings - and potentially saying that default uses hg38 seqnames (so 1,2,X rather than chr1, chr2, chrX)

Reply:
Good point! Or alternatively, we could convert integers to strings, which is the default in the R package. Another minor point is that we could consider naming it `SeqNameFilter` to be consistent with the R package

Reply:
> and potentially saying that default uses hg38 seqnames

Related to #7 and #11 

Maybe we could document this at the resource level?

> One minor failure mode of this filter is that you get a rather obscure error if you pass chromosomes as integers instead of strings
> Or alternatively, we could convert integers to strings

I ended up doing a conversion

> SeqNameFilter

Changed